recipes/piqs/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "piqs" %}
+{% set version = "1.2" %}
+{% set sha256 = "2be0f330ebf5cdad5c353c763d683054f09c6a9d08f8b34a71a056c29755cd75" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nathanshammah/piqs/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True   # [py2k]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python >=3.5
+    - pip
+    - numpy
+    - cython >=0.21
+  
+  run:
+    - python >=3.5
+    - {{ pin_compatible('numpy') }}
```

Comment:
Do we need the fancy build/host/run if we are using the new `pin_compatible` stuff?